src/MessageImage.tsx

```diff
@@ -59,7 +62,7 @@ export function MessageImage<TMessage extends IMessage = IMessage>({
         <Image
           {...imageProps}
           style={[styles.image, imageStyle]}
-          source={{ uri: currentMessage.image }}
+          source={{ uri: currentMessage.image, ...imageSourceProps }}
```

Comment:
Would the following change make more sense?  As it'll preserve the uri coming from the message and not allowed to be overridden. 

```suggestion
          source={{ ...imageSourceProps, uri: currentMessage.image }}
```

Reply:
Yep, updated!